core/runtime/pom.xml

```diff
@@ -156,7 +156,7 @@
                         <parentFirstArtifact>io.smallrye.common:smallrye-common-net</parentFirstArtifact>
                         <parentFirstArtifact>io.smallrye.common:smallrye-common-os</parentFirstArtifact>
                         <parentFirstArtifact>io.smallrye.common:smallrye-common-ref</parentFirstArtifact>
-                        <parentFirstArtifact>org.graalvm.sdk:graal-sdk</parentFirstArtifact>
+                        <parentFirstArtifact>org.graalvm.sdk:nativeimage</parentFirstArtifact>
```

Comment:
Could you drop `org.graalvm.sdk:native-image` a few lines below in the js section. Apparently I made a typo and it's not useful anymore.

Reply:
Done